src/xbot/common/command/BaseRobot.java

```diff
@@ -55,6 +57,7 @@ public BaseRobot() {
         super();
         setupInjectionModule();
         
+        pdp = injector.getInstance(PowerDistributionPanelWPIAdapter.class);
```

Comment:
This won't work here. It's gotta be in `initializeSystems` next to all the other things getting created.